### PR TITLE
Provide examples for quoting shell words

### PR DESCRIPTION
**Pull request by jkeenan**
Per suggestion by Andy Dougherty.

For: https://github.com/Perl/perl5/issues/18573

### CODE REVIEW

**Comment by jkeenan** [comment]
Updated p.r., per @Tux request.

File: INSTALL [hunk 1]
```diff
@@ -132,7 +132,31 @@ Configure supports a number of useful options.  Run
...
+    -Dcc=clang
+
+Sometimes the value to be supplied for a switch is a string which contains
+spaces.  In that case, the value needs to be quoted so as to delimit that
+"shell word" from any following switch.  Example:
+
+    sh ./Configure -des \
+        -Doptimize="-O2 -pipe -fstack-protector -fno-strict-aliasing" \
+        -Dusedevel
+
```
Comment (Tux) [inline INSTALL:148]: I'd advice to put `-des` last always, but I do not know why I would

**Review by Tux — APPROVED** [review]

File: Configure [hunk 2]
```diff
@@ -1834,8 +1834,8 @@ Usage: $me [-dehrsEKOSV] [-f config.sh] [-D symbol] [-D symbol=value]
   -r : reuse C symbols value if possible (skips costly nm extraction).
   -s : silent mode, only echoes questions and essential information.
   -D : define symbol to have some value:
-         -D symbol         symbol gets the value 'define'
-         -D symbol=value   symbol gets the value 'value'
+         -D symbol                symbol gets the value 'define'
+         -D symbol='some value'   symbol gets the 'some value'
```
Comment (Tux) [inline Configure]: Maybe `symbol is set to 'some value'` ? This reads silly

File: configure.com [hunk 3]
```diff
@@ -341,8 +341,8 @@ $   DECK
  "-r" : reuse C symbols value if possible (skips costly nm extraction).*
  "-s" : silent mode, only echoes questions and essential information.
  -"D" : define symbol to have some value:                              *
-         -"Dsymbol"         symbol gets the value 'define'
-         -"Dsymbol=value"   symbol gets the value 'value'
+         -"Dsymbol"             symbol gets the value 'define'
+         -"Dsymbol=some value"  symbol gets the value "some value"
```
Comment (Tux) [inline configure.com]: same here

**Review by Tux — CHANGES_REQUESTED** [review]

File: Configure (hunk 2 above)
Comment (jkeenan) [reply Configure]: > Maybe `symbol is set to 'some value'` ? This reads silly

@Tux, I'm a bit confused.  You appeared to like this wording [two days ago](https://github.com/Perl/perl5/issues/18573#issuecomment-779099615).  The objective here is simply to illustrate how to handle a value that contains a space.

File: INSTALL (hunk 1 above)
Comment (jkeenan) [reply INSTALL:148]: > I'd advice to put `-des` last always, but I do not know why I would

Well, I customarily start off with `-des -Dusedevel` ... but for the purpose of the example we need something after the quoted value.

File: Configure (hunk 2 above)
Comment (Tux) [reply Configure]: Your confusion is my fault. In my initial approval I only looked at the left side of the change.

`symbol gets the 'some value'

reads weird, that is why I would prefer the slight rewording to `symbol is set to 'some value'`
Very sorry for not paying more attention and causing noise

**Review by Tux — APPROVED** [review]

**Comment by Tux** [comment]
+1 Go for it!

**Pull request merged**